resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/KubernetesSuite.scala

```diff
@@ -109,7 +112,7 @@ private[spark] class KubernetesSuite extends SparkFunSuite
     runSparkPiAndVerifyCompletion()
   }
 
-  test("Run SparkPi with a master URL without a scheme.") {
+  test("Run SparkPi with a master URL without a scheme.", NoDCOS) {
```

Comment:
Mark this test as not appropriate for DC/OS as we run against insecure api server and when no scheme is defined current implementation will default to https. In the future I will remove this with targeting the secure api server on dc/os.